hdf5.spec

```diff
@@ -291,7 +286,9 @@ ln -s ../configure .
 %configure \
   %{configure_opts} \
   --enable-cxx \
-  --enable-java
+  --enable-java \
+  --without-examplesdir
```

Comment:
Why did this need to be added?

Reply:
This may have been an issue with my environment when running manually.  The examples wanted to be compiled/built in /usr/share.... but in my env I did not have privs.  I tried to move it but was not successful...  in order to make progress I removed the examples.    I can re-enable and see if it passes in CI

Reply:
we don't need the examples to be built so it's ok to disable them

Reply:
But we are trying to provide an update to what is on the distro so we want to strive to providing as compatible an update as possible.  We don't want to try to overcome substantial hurdles in doing so, but we also don't want to provide a "less than compatible" update for what are trivially resolvable issues.

Reply:
> The examples wanted to be compiled/built in /usr/share.... but in my env I did not have privs.

No RPM build environment should.  That's intentional as it catches broken RPM builds that are trying to install files outside of the `BUILDROOT` as this one appears to be doing.  This sounds like the examples are not honouring the `DESTDIR` that `make install` is invoked with.  This would be an upstream bug.

> I can re-enable and see if it passes in CI

It won't pass in CI either as it breaking is by design.  So we have to determine why the examples are not honouring `make install`'s  `DESTDIR`.

Reply:
I'd still like us to not introduce this packaging regression if fixing it is a simple matter, which it likely is.  It's probably just a case of a DESTDIR having been dropped from the examples `install` target in it's `Makefile`.

If you can't find that quickly, let me know and I will take a look and see how quickly/easily this can be fixed.